src/test/java/tests/AddRemoveTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import io.qameta.allure.Description;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class AddRemoveTest extends BaseTest{
+        private static final Logger logger = LoggerFactory.getLogger(AddRemoveTest.class);
+
+        @Test
+        @Description("Verify user can click add and remove button")
+        public void clickAdd() throws InterruptedException, IOException {
+        try {
+            logger.info("The user is on homepage");
+            String homepageHeading = homePage.getHomePageHeading();
+            Assert.assertEquals(homepageHeading, "Welcome to the-internet");
+            homePage.clickAdd();
+            Assert.assertTrue(addRemovePage.isAddElementButtonVisible());
+            logger.info("Test passed");
```

Comment:
Only use the Test passed log message when the entire test case passes. In this file Test passed log message is used on multiple lines, which is confusing